scripts/diffmazing_cropper.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+#
+# Converts diff images from Cypress into snapshots we can commit to Git.
+
+
+if [ $# -eq 0 ]
+then
```

Comment:
You can avoid repeated `echo` commands using multiline strings (here and elsewhere):
```
cat <<__END__
Usage:

  $0 <diff files to convert>
  
  Example:
    $0 ~/downloads/*.diff.png

__END__
```

Reply:
I actually find multiple echo's cleaner, as it allows me to keep the same indentation.

This:
```
if [ $# -eq 0 ]
then
  echo ""
  echo "Usage"
  echo "  $0 <diff files to convert>"
  echo ""
  echo "Example"
  echo "  $0 ~/downloads/*.diff.png"
  echo ""
  exit -1
fi
```

Looks better (to me) than this:
```
if [ $# -eq 0 ]
then
  cat <<__END__
Usage
  $0 <diff files to convert>

Example
  $0 ~/downloads/*.diff.png

__END__
  exit -1
fi
```

Reply:
Will leave this one unresolved. LMK if you want to discuss further!

Reply:
SGTM